python/gtsam_py/python/tests/test_SimpleCamera.py

```diff
@@ -5,7 +5,7 @@
 
 See LICENSE for the license information
 
-PinholeCameraCal3_S2 unit tests.
+SimpleCamera unit tests.
```

Comment:
SimpleCamera should no longer be there.

Reply:
@ProfFan @dellaert should this file just be removed then?